composer.json

```diff
@@ -22,6 +22,7 @@
         "laravel/socialite": "^5.12",
         "laravel/tinker": "^2.8",
         "php-open-source-saver/jwt-auth": "^2.3",
+        "reliese/laravel": "^1.3",
```

Comment:
This seems to be a development related package and we already use the `barryvdh/laravel-ide-helper` package which can generate model properties in docblocks.

Reply:
The tests are failing as well.